website/docs/r/container_ingress_secret_opaque.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "Kubernetes Service"
+layout: "ibm"
+page_title: "IBM: ibm_container_ingress_secret_opaque"
+description: |-
+  Registers an IBM Cloud Secrets Manager certificate secret with your cluster
+---
+
+# ibm_container_ingress_secret_opaque
+Registers an IBM Cloud Secrets Manager secret type certificate with your IBM Cloud Kubernetes Service or Red Hat OpenShift on IBM Cloud cluster. For more information about how opaque secrets can be used see [about Secrets Manager secrets](https://cloud.ibm.com/docs/containers?topic=containers-secrets#non-tls)
+
+## Example usage
+
+```terraform
+resource "ibm_container_ingress_secret_opaque" "secret" {
+  cluster="exampleClusterName"
+  secret_name="mySecretName"
+  secret_namespace="mySecretNamespace"
+  cert_crn="cert:region:crn"
```

Comment:
cert_crn is required /optional argument. its not defined in below section 
`Argument reference`